tests/security/cc/syscalls.pm

```diff
@@ -0,0 +1,57 @@
+# Copyright (C) 2021 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: Run "syscalls" test case of "audit-test" test suite
+# Maintainer: liuxiaojing <xiaojing.liu@suse.com>
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use audit_test;
+
+sub run {
+    my ($self) = shift;
+
+    select_console "root-console";
+    assert_script_run('sed -i \'/\[Unit\]/a\StartLimitIntervalSec=0\' /usr/lib/systemd/system/auditd.service');
+    assert_script_run("systemctl daemon-reload");
+    assert_script_run("sed -i 's/-a task,never/#-a task,never/' /etc/audit/rules.d/audit.rules");
+    assert_script_run('systemctl restart auditd.service');
```

Comment:
These 4 lines needs to be setup for syscall testing, right? Although I am not sure if it needs to be applied all tests for the common configuration. Because it shows this is the workaround from document.

Reply:
Not sure if it is needed for other tests, adding a parameter to control that.

Reply:
Good to know, I think use the parameter to control would be a good solution. :)